### PR TITLE
Upgrade OpenLP.app to v2.2

### PR DESCRIPTION
**Pull request by joelbcastillo**
Upgrading OpenLP to version 2.2 from version 2.05

### CODE REVIEW

File: Casks/openlp.rb [hunk 1]
```diff
@@ -1,9 +1,8 @@
 cask :v1 => 'openlp' do
-  version '2.0.5'
-  sha256 '0ba5a37c359394c277e221f8ca74191d3425cf3cfbd70699d81346c77c93d746'
+  version '2.2'
+  sha256 :no_check
 
-  # sourceforge.net is the official download host per the vendor homepage
-  url "http://downloads.sourceforge.net/project/openlp/openlp/#{version}/OpenLP-#{version}.dmg"
+  url "https://get.openlp.org/2.2/OpenLP-2.2.dmg"
```
Comment (fanquake) [inline Casks/openlp.rb]: Use #{version} here

File: Casks/openlp.rb [hunk 2]
```diff
@@ -1,9 +1,8 @@
 cask :v1 => 'openlp' do
-  version '2.0.5'
-  sha256 '0ba5a37c359394c277e221f8ca74191d3425cf3cfbd70699d81346c77c93d746'
+  version '2.2'
+  sha256 :no_check
```
Comment (fanquake) [inline Casks/openlp.rb]: Please leave the shasum

**Comment by adityadalal924** [comment]
Thank you for the contribution. It needed some fixes, so they were made in a1c0286aab4691ef6c73240b78fddac30c07370a. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by adityadalal924** [comment]
@joelbcastillo Made the appropriate fixes, but @fanquake is absolutely correct.

Thanks!

**Pull request closed without merging**